src/alchemlyb/workflows/abfe.py

```diff
@@ -0,0 +1,675 @@
+                     ax=None):
+        '''Plot the dHdl for TI estimator using
+        :func:`~alchemlyb.visualisation.plot_ti_dhdl`.
+
+        Parameters
+        ----------
+        dhdl_TI : str
+            The filename for the plot of TI dHdl. Default: 'dhdl_TI.pdf'
+        labels : List
+            list of labels for labelling all the alchemical transformations.
+        colors : List
+            list of colors for plotting all the alchemical transformations.
+            Default: ['r', 'g', '#7F38EC', '#9F000F', 'b', 'y']
+        ax : matplotlib.axes.Axes
+            Matplotlib axes object where the plot will be drawn on. If ax=None,
+            a new axes will be generated.
+
+        Returns
+        -------
+        matplotlib.axes.Axes
+            An axes with the TI dhdl drawn.
+        '''
+        self.logger.info('Plot TI dHdl.')
+        if 'ti' in self.estimator:
+            ax = plot_ti_dhdl(self.estimator['ti'], units=self.units,
+                              labels=labels, colors=colors, ax=ax)
+            ax.figure.savefig(join(self.out, dhdl_TI))
+            self.logger.info(f'Plot TI dHdl to {dhdl_TI} under {self.out}.')
+            return ax
+        else:
+            raise ValueError('No TI data available in estimators.')
+
+    def plot_dF_state(self, dF_state='dF_state.pdf', labels=None, colors=None,
+                      orientation='portrait', nb=10):
+        '''Plot the dF states using
+        :func:`~alchemlyb.visualisation.plot_dF_state`.
+
+        Parameters
+        ----------
+        dF_state : str
+            The filename for the plot of dF states. Default: 'dF_state.pdf'
+        labels : List
+            list of labels for labelling different estimators.
+        colors : List
+            list of colors for plotting different estimators.
+        orientation : string
+            The orientation of the figure. Can be `portrait` or `landscape`
+        nb : int
+            Maximum number of dF states in one row in the `portrait` mode
+
+        Returns
+        -------
+        matplotlib.figure.Figure
+            An Figure with the dF states drawn.
+        '''
+        self.logger.info('Plot dF states.')
+        fig = plot_dF_state(self.estimator.values(), labels=labels, colors=colors,
+                            units=self.units,
+                            orientation=orientation, nb=nb)
+        fig.savefig(join(self.out, dF_state))
+        self.logger.info(f'Plot dF state to {dF_state} under {self.out}.')
+        return fig
+
+    def check_convergence(self, forwrev, estimator='autombar', dF_t='dF_t.pdf',
+                     ax=None):
+        '''Compute the forward and backward convergence using
+        :func:`~alchemlyb.convergence.forward_backward_convergence`and
+        plotted with
+        :func:`~alchemlyb.visualisation.plot_convergence`.
+
+        Parameters
+        ----------
+        forwrev : int
```

Comment:
So the `forwrev` will also be a keyword in run(). 

I do agree that for check_convergence(), the keyword `n_timepoints` is more clear, but it will be a bit confusing if we put `n_timepoints` as a keyword in run(). 
It would also be confusing if we have two different keywords to mean the same thing in run() and check_convergence().